### ISSUE TITLE
Documentation: .filters description doesn't make sense

### ISSUE THREAD

**Issue by cwilso** [mrdoob/three.js#13666]
The description of the .filters on Audio doesn't make sense -
 just says "Whether the audio is currently playing. Default is empty array."  rather than describing what an element in the array is or represents (WebAudio BiquadFilterNode or any chainable filter, I guess?).  Also, the setFilter behavior seems like it would blow away any previous filters.

(around this line: https://github.com/mrdoob/three.js/blob/683289bd00dc2eb8230744c03d7e7c67c68a2f2b/docs/api/audio/Audio.html#L66)

**Comment by Mugen87** [comment]
Yeah, i guess the current description of `.filters` is some sort of copy/paste error. How about:
```
Represents an array of [link:https://developer.mozilla.org/en-US/docs/Web/API/BiquadFilterNode BiquadFilterNode]s. Can be used to apply a variety of low-order filters to create more complex sound effects.
```
> Also, the setFilter behavior seems like it would blow away any previous filters.

Yes, the description is wrong, too. The method does not perform an add operation. It just sets a single filter and thus overwrites existing filters.

**Comment by Mugen87** [comment]
Would you like to do a PR with the changes 😇

**Comment by cwilso** [comment]
What's the right thing to do here?

1. Describe current behavior of setFilter and setFilters, and the .filters array.  This will have some odd text, since the setFilter* methods are kind of odd.  ("This is an array containing any filters that have been set.")
2. Quietly remove mention of setFilters, change setFilter description to be correct. ("inserts a Web Audio filtering node into the audio output.  Takes a Web Audio AudioNode, such as BiquadFilterNode. Will replace any other filter set.")
3. As #2, but remove setFilters code too.
4. Fix setFilters code?  This is pretty open-ended; I expect you'd really want to have a setFilter that took an offset, and setFilters shouldn't exist.

I'm not really sure?  TBH, I'm not sure what value this is trying to add over the Web Audio API, since it's still using the Web Audio nodes (and the developer has to understand them).

**Comment by Mugen87** [comment]
As a first step, i would choose option one in order to ensure the descriptions actually reflect the current implementation. This is something that we can do now.

However, refactoring `.setFilter()` and `. setFilters()` is something that we should consider. As you said, a single `.setFilter()` method with different semantics is maybe sufficient. Let's see what @mrdoob thinks about it.

**Comment by mrdoob** [comment]
It's tempting to go with `addFilter()`, `removeFilter()`, `addFilterAt()`, `removeFilterAt()`, ... but then we'll be doing yet another graph 😅
